# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/RemoteAccessibleBase.h
# Commit: b92893ebb007
# Full Hash: b92893ebb007f76bc8b7a9c070e2938a0598a381
# Author: James Teh <jteh@mozilla.com>
# Date: 2021-10-08 09:48:33
# Regressor Bug: 1734322
# File Overlap Count: 1
# Description:
#   Bug 1734322: Cash DOM node id in the parent process a11y cache. r=eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D127757
# ==============================================================================

diff -r 92bfb5c41eb7 -r b92893ebb007 accessible/ipc/RemoteAccessibleBase.h
--- a/accessible/ipc/RemoteAccessibleBase.h	Fri Oct 08 00:52:39 2021 +0000
+++ b/accessible/ipc/RemoteAccessibleBase.h	Fri Oct 08 01:57:46 2021 +0000
@@ -223,6 +223,12 @@
                          : nullptr;
   }
 
+  /**
+   * Return the id of the dom node this accessible represents.  Note this
+   * should probably only be used for testing.
+   */
+  virtual void DOMNodeID(nsString& aID) const;
+
   // HyperTextAccessibleBase
   // XXX Implement this once it's cached.
   virtual int32_t CaretOffset() const override { return -1; }